pkg/cmd/generate.go

```diff
@@ -28,7 +28,12 @@ import (
 	"k8s.io/client-go/pkg/runtime"
 )
 
-func Generate(files []string) error {
+func Generate(paths []string) error {
+
+	files, err := GetAllYAMLFiles(paths)
```

Comment:
@kadel hmm, makes sense to move both `ifFilesPassed` and `GetAllYAMLFiles` out of cmd, but that's for another PR. Tracking in #185